README.md

```diff
@@ -854,7 +854,7 @@ See [strava-oauth-token](/bin/strava-oauth-token) or [strava-ruby-cli](https://g
 
 ### Webhooks
 
-Strava provides a [Webhook Event API](https://developers.strava.com/docs/webhooks/) that requires special access obtained by emailing [developers@strava.com](mailto:developers@strava.com).
+Strava provides a [Webhook Event API](https://developers.strava.com/docs/webhooks/) that is now turned on by default for all applications.
```

Comment:
Thanks! Feels like we don't need to say "that is now turned on ..." anymore? It's the default.

Reply:
Makes sense to me! I'll whack the end.